frontends/common/constantFolding.h

```diff
@@ -162,6 +162,23 @@ class ConstantFolding : public PassManager {
         if (typeMap != nullptr) passes.push_back(new ClearTypeMap(typeMap));
         setName("ConstantFolding");
     }
+
+    /** Function definition for hook to filter which Expression objects are processed.
```

Comment:
Optional nit: Even in this class the comments are mixed but at least on the open-source side we have been increasingly using three slashes `///`.

Reply:
I'm happy to switch to `///`